src/ua.c

```diff
@@ -739,6 +740,8 @@ static int uri_complete(struct ua *ua, struct mbuf *buf, const char *uri)
 {
 	size_t len;
 	int err = 0;
+	int uri_is_ip = 0;
```

Comment:
it is probably better to use a `bool` type here since the value
is either `true` or `false`.